nitter.conf

```diff
@@ -5,5 +5,5 @@ title = "nitter"
 staticDir = "./public"
 
 [Cache]
-directory = "/tmp/niter"
-profileMinutes = 10  # how long to cache profiles
\ No newline at end of file
+directory = "./tmp"
```

Comment:
For security purposes, I think firejail is better suited for this. It's in `/tmp/` to keep everything in RAM, reducing disk load. I'm not an expert on this topic so please let me know if you think I'm wrong.

Reply:
Firejail is not installed by default on all systems. Next to this `/tmp` is not RAM, it just will be flushed on a reboot. `/dev/shm` would be real RAM.

Reply:
Fair enough, thank you.